autoware_launch/launch/e2e_simulator.launch.xml

```diff
@@ -31,15 +33,30 @@
   <!-- System -->
   <arg name="launch_system_monitor" default="false" description="launch system monitor"/>
   <arg name="launch_dummy_diag_publisher" default="false" description="launch dummy diag publisher"/>
-  <arg
-    name="system_error_monitor_param_path"
-    default="$(find-pkg-share autoware_launch)/config/system/system_error_monitor/system_error_monitor.awsim.param.yaml"
-    description="system error monitor param path"
-  />
+
   <!-- Tools -->
   <arg name="rviz" default="true" description="launch rviz"/>
   <arg name="rviz_config" default="$(find-pkg-share autoware_launch)/rviz/autoware.rviz" description="rviz config"/>
 
+  <group scoped="false" if="$(eval '\'$(var simulator_type)\' == \'carla\'')">
+    <include file="$(find-pkg-share carla_autoware)/carla_autoware.launch.xml">
+      <arg name="carla_map_name" value="$(var carla_map)"/>
+    </include>
```

Comment:
What is the purpose of the file carla_autoware.launch.xml?
What is the related PR if necessary?

Reply:
`carla_autoware.launch.xml` will launch the nodes to communicate between CARLA and Autoware.

Reply:
I see
The following code seems to start the same file regardless of whether it is awsim or carla. Is it necessary to branch with an if statement?

```
    <arg
      name="system_error_monitor_param_path"
      default="$(find-pkg-share autoware_launch)/config/system/system_error_monitor/system_error_monitor.awsim.param.yaml"
      description="system error monitor param path"
    />
```